secure-boot-recovery5/README.md

```diff
@@ -62,7 +62,15 @@ mkdir -p metadata
 ../rpiboot -d . -j metadata
 ```
 
+## Requirement for flashed OTP
+The BCM2712 will not boot a signed EEPROM image unless it holds the public key in its One Time Programmable (OTP) memory. If you try to boot a signed EEPROM image without burning the public key into OTP, the boot LED on the Raspberry Pi 5B will display an error code by flashing green 2 times.
```

Comment:
I think this first paragraph is only true on a BCM2712 C1, unfortuantely BCM2712 D0 (CM5, Pi500 and newer Pi5) cannot do the flash code from the bootrom.